package.json

```diff
@@ -7,18 +7,26 @@
   "license": "UPL",
   "private": true,
   "scripts": {
-    "build": "webpack --config webpack.prod.js && npm run test",
+    "build": "npm-run-all -l build:notests test",
     "build:notests": "webpack --config webpack.prod.js",
     "clean": "rimraf dist",
     "start:dev": "webpack serve --hot --color --progress --config webpack.dev.js",
     "test": "jest --maxWorkers=50% --coverage=true",
     "test:ci": "jest --maxWorkers=50%",
-    "eslint": "eslint --ext .tsx,.js ./src/",
+    "eslint": "npm-run-all eslint:check",
+    "eslint:check": "eslint --ext .ts,.tsx,.js ./src/",
+    "eslint:apply": "eslint --ext .ts,.tsx,.js --fix ./src/",
     "license:check": "license-check-and-add check -f license-config.json",
-    "lint": "npm run license-check-and-add && npm run eslint",
-    "format": "prettier --check --write ./src/**/*.{tsx,ts} && license-check-and-add add -f license-config.json",
+    "license:apply": "license-check-and-add add -f license-config.json",
+    "lint": "npm-run-all -l -p format:check eslint",
```

Comment:
I think we can use `format` for short?

Reply:
`format` is already the alias for checking licenses and code style (indentation, whitespaces, etc). This one does that plus linting, which is the next step and kinda analogous to `spotbugs:check` on the main repo.

Reply:
oh sorry I mean within the command, we can use `format` instead of `format:check` for short :D

Reply:
Oh, yea, that works but I just find it neater to not use `npm-run-all foo` where `foo` also starts with invoking `npm-run-all`. It doesn't really matter too much, though.